roles/app/templates/application.yml.j2

```diff
@@ -88,9 +88,3 @@ GEOCODER_SERVICE: {{ geocoder_service }}
 {% if geocoder_timeout is defined %}
 GEOCODER_TIMEOUT: {{ geocoder_timeout }}
 {% endif %}
-
-{{ custom_application_env_vars | default('') }}
```

Comment:
This is still used in some cases.

Reply:
Is that in secrets? I couldn't find any use in ofn-install.

Reply:
Yep :+1: